types/hapi/v17/definitions/server/server-options.d.ts

```diff
@@ -0,0 +1,188 @@
+import * as http from "http";
+import * as https from "https";
+import * as catbox from "catbox";
+import {MimosOptions} from "mimos";
+import {PluginSpecificConfiguration, RouteOptions, ServerOptionsCache} from "hapi";
+
+/**
+ * he server options control the behavior of the server object. Note that the options object is deeply cloned
+ * (with the exception of listener which is shallowly copied) and should not contain any values that are unsafe to perform deep copy on.
+ * All options are optionals.
+ * [See docs](https://github.com/hapijs/hapi/blob/master/API.md#-server-options)
+ */
+export interface ServerOptions {
```

Comment:
Maybe one test where all the options of this interface is used, would be nice?

Reply:
@BorntraegerMarc very good idea. I created this [test file](https://github.com/rafaelsouzaf/DefinitelyTyped/blob/6a92487939470e96f6f3219963e66962a97ff6ac/types/hapi/v17/test/server/server-options.ts).